mobile/src/main/java/com/gelakinetic/mtgfam/fragments/FamiliarFragment.java

```diff
@@ -321,7 +321,8 @@ public void receiveTutorCardsResult(long multiverseId) {
      * Override this to receive results from ResultListDialogFragments
      *
      * @param orderByStr The sort order string
+     * @param dontSort   if this is true, don't sort anything
      */
-    public void receiveSortOrder(String orderByStr) {
+    public void receiveSortOrder(String orderByStr, boolean dontSort) {
```

Comment:
The abbreviations in the variable names are harder to read than a full e.g. doNotSort. The same goes for some tags.